docs/Projects/TAD/tools_comparison.md

```diff
@@ -0,0 +1,228 @@
+# Comparison of tools for transparency of algorithmic decision making
+
+In [tools.md](tools.md) we have researched a few tools which we want to investigate further, this document is the next
+step in that investigation. The checklist we will lay these tools against is written down at [index.md](index.md). The
+Fulfilled column will give a numerical value based on the priority which can then be compared with the other tools per
+section. The scoring is as follows: {M:4, S:3, C:2, W:1}.
+
+## Summary of the comparison
+
+| Tool                     | Requirement             | Score |
+|:-------------------------|:------------------------|:------|
+| AIVerify                 | Functionality           | 36    |
+|                          | Reliability             | 13    |
+|                          | Usability               | 9.4   |
+|                          | Help & Documentation    | 2.8   |
+|                          | Performance Efficiency  | 9     |
+|                          | Maintainability         | 15.8  |
+|                          | Security                | 8.3   |
+|                          | Compatibility           | 10.4  |
+|                          | Accessibility           | 0     |
+|                          | Portability             | 7.5   |
+|                          | Deployment              | 1.5   |
+|                          | Legal & Compliance      | 19    |
+| VerifyML                 | Functionality           |       |
```

Comment:
Couldn't we make VerifyML a second column (next to the one already filled for AIVerify)?

Reply:
I was hoping we could do that also for the larger tables below

Reply:
Then the possibility for comments would not be there though, and i think that is more valuable then a specific number

Reply:
I want to agree with you, but for me it just is too large of a table to have it all next to each other with the comments :S